src/auth/reference_modules/requirements.txt

```diff
@@ -4,6 +4,6 @@ requests==2.32.2
 ldap3==2.6
 pyyaml==6.0.1
 python3-saml==1.16.0
-lxml==5.2.1
+lxml==6.0.0
```

Comment:
@imilinovic are those modules manually tested? -> Do we have to test something manually?